Xamarin.Essentials/AppInfo/AppInfo.ios.tvos.watchos.cs

```diff
@@ -23,5 +23,23 @@ static string GetBundleValue(string key)
         static void PlatformShowSettingsUI() =>
             throw new FeatureNotSupportedException();
 #endif
+
+#if __IOS__ || __TVOS__
+        static AppTheme PlatformRequestedTheme()
+        {
+            if (!Platform.HasOSVersion(13, 0))
+                return AppTheme.Unspecified;
```

Comment:
shouldn't it be AppTheme.Light? since it is what the OS used pre-13

Reply:
That is a good question. I mean technically there was no "theme" at all.... @redth?

Reply:
I think unspecified is a sane default here since pre 13 there really wasn't a theme like James said, it was just... the theme.